sass/base/_typography.scss

```diff
@@ -112,16 +84,28 @@ html[lang="ar"] {
   }
 }
 
-// @TODO ZH might need special attention.
-html[lang="ru"],
-html[lang="zh"] {
+html[lang="ru"] {
   h1,
   h2,
   h3,
   h4,
   h5,
   h6 {
     font-family: $cd-font--noto;
+    font-weight: 400;
+  }
+}
+
+// @TODO ZH might need special attention.
+html[lang="zh-hans"] {
```

Comment:
in the `_cd-resets.css` it's still `lang="zh"` and it's not using the correct Chinese font I think: https://github.com/UN-OCHA/common_design/blob/5f70fdc64f4bdb34214b3ef3c667eb90691c949a/sass/cd/_cd-resets.scss#L44